src/browser.ts

```diff
@@ -61,6 +61,19 @@ export class GrpcClient {
   auth: GoogleAuth;
   promise?: PromiseConstructor;
 
+  /**
+   * Browser version of GrpcClient
+   * A class which keeps the context of gRPC and auth for the gRPC.
```

Comment:
Line 66 does not apply here - there is no gRPC here at all :) Let's say something like, `Implements GrpcClient API for a browser using grpc-fallback protocol (sends serialized protobuf to HTTP/1 $rpc endpoint).`